contrib/postprocessing/lethe_pyvista_tools.py

```diff
@@ -152,13 +247,14 @@ def sort_by_array(self, reference_array_name):
             pbar.update(1)
 
     # Creates or modifies array
-    def array_modifier(self, reference_array_name = "ID", array_name = "new_array", restart_array = False,  condition = "", array_values = 0, standard_value = 0, reference_time_step = 0, time_dependent = False):
+    def modify_array(self, reference_array_name = "ID", array_name = "new_array", restart_array = False,  condition = "", array_values = 0, standard_value = 0, reference_time_step = 0, time_dependent = False):
```

Comment:
Also, I would suggest to have the same default value for `array_name` as the functions for mixing index, otherwise we have to make sure to rename one of them (`new_array` vs `particle_color`)